### PR TITLE
Added jquery.photoswipe library

### PR DESCRIPTION
**Pull request by brandomeniconi**

### CODE REVIEW

File: files/jquery.photoswipe/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
```
Comment (jimaek) [inline files/jquery.photoswipe/update.json]: No tags here https://github.com/yaquawa/jquery.photoswipe/releases
the bot will fail

File: files/jquery.photoswipe/info.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author = "Wang Shugen"
+github = "https://github.com/yaquawa/jquery.photoswipe"
+homepage = "https://github.com/yaquawa/jquery.photoswipe"
+description = "The jQuery plugin for PhotoSwipe"
+mainfile = "dist/jquery.photoswipe.js"
```
Comment (jimaek) [inline files/jquery.photoswipe/info.ini]: Just `jquery.photoswipe.js`

**Comment by brandomeniconi** [comment]
fixed as suggested

**Comment by brandomeniconi** [comment]
it's ok now @jimaek ? does I need to resubmit the PR? also #9713 :)

File: files/jquery.photoswipe/update.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "jquery.photoswipe",
+  "files": {
+    "basePath": "dist/",
```
Comment (jimaek) [inline files/jquery.photoswipe/update.json]: change the basepath to `dist/min/` and it should be good to go

**Comment by brandomeniconi** [comment]
sorry @jimaek! fixed it

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**